common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1926,6 +1926,20 @@ def filestore(self):
         )
         return self.resources_fs
 
+    @property
+    def field_data(self):
```

Comment:
I believe this is not needed as the Runtime class also considers this as a deprecated field:
https://github.com/openedx/XBlock/blob/288f01476bc571664d6a8ed990528ca3341e6bb4/xblock/runtime.py#L577-L596

I'll remove it from here!

Reply:
Done!